frontend/src/App.js

```diff
@@ -1,11 +1,16 @@
+import React from 'react'
```

Comment:
@JoeDravarol is this necessary? I noticed it was not in the Main branch

Reply:
That looks correct. It's for rendering JSX (Similar to EJS but for React).